components/python/mergedeep/Makefile

```diff
@@ -0,0 +1,32 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# This file was automatically generated using the following command:
+#   $WS_TOOLS/python-integrate-project mergedeep
+#
+
+BUILD_STYLE = setup.py
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME =		mergedeep
+COMPONENT_VERSION =		1.3.4
+COMPONENT_SUMMARY =		mergedeep - A deep merge function for 🐍.
```

Comment:
This non-ascii character in the summary may be the offender in the server issue. @mtelka @AndWac